pkgs/development/ocaml-modules/hc/default.nix

```diff
@@ -0,0 +1,27 @@
+{
+  lib,
+  buildDunePackage,
+  fetchurl,
+}:
+
+buildDunePackage rec {
+  pname = "hc";
+  version = "0.5";
+
+  # upstream git server is misconfigured and cannot be cloned
+  src = fetchurl {
+    url = "https://git.zapashcanon.fr/zapashcanon/hc/archive/${version}.tar.gz";
```

Comment:
Is it safe to use `fetchurl` with this URL? `fetchzip` might be more adequate.

Reply:
I'm not familiar with the safety of `fetchurl` do you have anything for me to read?